uefi/src/lib.rs

```diff
@@ -74,8 +77,48 @@
 //! only unfold their potential when you invoke `uefi::helpers::init` as soon
 //! as possible in your application.
 //!
+//! # Discuss and Contribute
+//!
+//! For general discussions, feel free to join us in our [Zulip] and ask
+//! your questions there.
+//!
+//! Further, you can submit bugs and also ask questions in our [issue tracker].
+//! Contributions in form of a PR are also highly welcome. Check our
+//! [contributing guide][contributing] for details.
+//!
+//! # MSRV
+//! <!-- Keep in Sync with README! -->
+//!
+//! The minimum supported Rust version is currently 1.70.
+//! Our policy is to support at least the past two stable releases.
+//!
+//! # License
+//! <!-- Keep in Sync with README! -->
+//!
+//! The code in this repository is licensed under the Mozilla Public License 2.
+//! This license allows you to use the crate in proprietary programs, but any
+//! modifications to the files must be open-sourced.
+//!
+//! The full text of the license is available in the [license file][LICENSE].
+//!
+//! # Trivia and Background
+//!
+//! [UEFI] started as the successor firmware to the BIOS in x86 space and
+//! developed to a universal firmware specification for various platforms, such
+//! as ARM. It provides an early boot environment with a variety of
+//! [specified][spec] ready-to-use "high-level" functionality, such as accessing
+//! disks or the network. EFI images, the files that can be loaded by an UEFI
+//! environment, can leverage these abstractions to extend the functionality in
+//! form of additional drivers, OS-specific bootloaders, or any different kind
+//! of low-level applications.
+//!
+//! Even joke projects such as an [IRC client][uefirc] leveraging only existing
+//! UEFI boot services are possible! 😉
```

Comment:
I'd rather leave the IRC client out of the crate docs. (Especially since it's not currently maintained, I wouldn't want to give the impression that users should look there to see how to call uefi-rs.)

As an alternative, I wouldn't be opposed to putting it in the readme if we added a showcase section, e.g. "Projects that use uefi-rs".

Reply:
I just drop the entire paragraph. It is irrelevant. We are already referring to the UEFI spec. This is sufficient